src/system_metrics/service_mongoose_system_metrics.erl

```diff
@@ -208,3 +212,7 @@ msg_accept_terms_and_conditions() ->
     "      https://esl.github.io/MongooseDocs/latest/operation-and-maintenance/System-Metrics-Privacy-Policy/ \n"
     "- MongooseIM GitHub page - https://github.com/esl/MongooseIM \n"
     "The last sent report is also written to a file ~s">>.
+
+-spec exometer_loaded() -> boolean().
+exometer_loaded() ->
+    lists:keymember(exometer_core, 1, application:loaded_applications()).
```

Comment:
Wouldn't it be better to check if exometer instrumentation is enabled, i.e.
```suggestion
    case mongoose_config:lookup_opt([instrumentation, exometer]) of ...
```
This way, if `exometer` dies, but should be running, we would get the error - as we should. I think it's better to relay on the config as the source of truth.

Reply:
https://www.erlang.org/docs/28/apps/kernel/application.html#start/2

```
Argument Type specifies the type of the application. If omitted, it defaults to temporary.

   • If a permanent application terminates, all other applications and the entire Erlang node are also terminated.
```